auto_labeling_pipeline/labels.py

```diff
@@ -48,7 +48,7 @@ class SequenceLabels(Labels):
     def remove_overlapping(self) -> 'Labels':
         target = self.label_class(start_offset=0, end_offset=0, label='')
         labels = sorted(self.labels, key=operator.attrgetter('start_offset'))
-        labels = [target := label for label in labels if not label.overlap_with(target)]  # type: ignore
+        labels = [label for label in labels if not label.overlap_with(target)]  # type: ignore
```

Comment:
This is not correct. So it fails testing. If we expand the algorithm, it will be the following:

```python
_labels = []
for label in labels:
    if label.overlap_with(target):
        continue
    target = label  # type: ignore
    _labels.append(label)
```

Reply:
@Hironsan could provide a correct patch, please? We have a syntax error now.

Reply:
OK, Pushed.